ext/strings_test.go

```diff
@@ -329,3 +330,247 @@ func TestStringsWithExtension(t *testing.T) {
 		t.Fatalf("env.Extend(Strings()) failed: %v", err)
 	}
 }
+
+func TestStringFormat(t *testing.T) {
```

Comment:
Support+tests for CEL's date, time and duration types are still needed, I believe.

Reply:
Fixed - I've added support/tests for `Timestamp` and `Duration`.